Keen/Dataset/Dataset.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Keen.Core.Dataset
+{
+    using Query;
```

Comment:
Reminds me we need to start creating some guidelines around things like `using` inside and outside of namespaces and configure some stylecop rulesets: https://github.com/DotNetAnalyzers/StyleCopAnalyzers